products/llbuildSwift/CMakeLists.txt

```diff
@@ -38,19 +59,19 @@ if (SWIFTC_FOUND)
   endif()
 
   # Install both libllbuild and libllbuildSwift.
-  list(APPEND LLBUILD_LIBRARIES "${LLBUILD_LIBRARY_OUTPUT_INTDIR}/libllbuild.${DYLIB_EXT}")
-  list(APPEND LLBUILD_LIBRARIES "${LLBUILD_LIBRARY_OUTPUT_INTDIR}/libllbuildSwift.${DYLIB_EXT}")
+  list(APPEND LLBUILD_LIBRARIES "${LLBUILD_LIBRARY_OUTPUT_INTDIR}/${LIBLLBUILD_NAME}${CMAKE_SHARED_LIBRARY_SUFFIX}")
```

Comment:
We do explicitly pass `SHARED` to those libraries right?

Reply:
Yeah for both.